doc/buying_things_for_the_lab.md

```diff
@@ -1,10 +1,25 @@
-# Lorem ipsum dolores
+# Purchasing guidelines in the lab
 
 
-## sit amed, consectetur adipiscing elit. Sed vestibulum semper felis quis accumsan.
+## Buying a computer
 
 
+- Step 1: Before proceeding, discuss your computer needs and financial options with Olivier.
+- Step 2: Choose your desired specifications and options from the appropriate website (e.g., Apple).
+- Step 3: Save a PDF of the detailed quote or invoice (with all specifications) and send it to Roland Jaumain (roland.jaumain@uclouvain.be). He will look for available university discounts and provide a final invoice to you and Olivier.
+- Step 4: After acceptance, upload the final invoice to the Lab Drive folder under Main/Computers/Computers, to keep track of all computer purchases.  

```

Comment:
```suggestion
- Step 4: After acceptance, upload the final invoice to the Lab Drive folder under [Main/Computers/Computers](https://drive.google.com/drive/folders/1VsR3GPradIYer0yPuEBcIN9wFDStPiS9?usp=sharing), to keep track of all computer purchases.  

```

Reply:
the link here should be safe since it is created with the "people with access" restriction